internal/pkg/util/yamlprotoconverter_test.go

```diff
@@ -0,0 +1,271 @@
+package yamlprotoconverter_test
+
+import (
+	. "github.com/envoyproxy/xds-relay/internal/pkg/util"
```

Comment:
shouldn't need this import

Reply:
we need this to have access to `FromYAMLToProto`, no?

Reply:
nvm, the package name for tests was wrong.